guides/01_getting-started/03_sharing-your-app.md

```diff
@@ -77,20 +79,41 @@ fetch("https://pypi.org/pypi/gradio/json"
 });
 </script>
 
+You can see examples of web components look <a href="https://www.gradio.app">on the Gradio landing page</a>.
+
+You can also customize the appearance and behavior of your web component with attributes that you pass into the `<gradio-app>` tag:
+
+* `src`: as we've seen, the `src` attributes links to the URL of the hosted Gradio demo that you would like to embed
+* `space`: an optional shorthand if your Gradio demo is hosted on Hugging Face Space. Accepts a `username/space_name` instead of a full URL. Example: `gradio/Echocardiogram-Segmentation`. If this attribute attribute is provided, then `src` does not need to be provided.
+* `control_page_title`: a boolean designating whether the html title of the page should be set to the title of the Gradio app (by default `"false"`)
+* `initial_height`: the intial height of the web component while it is loading the Gradio app, (by default `"300px"`). Note that the final height is set based on the size of the Gradio app.
+* `container`: whether to show the border frame and information about where the Space is hosted (by default `"true"`)
+* `info`: whether to show just the information about where the Space is hosted underneath the embedded app (by default `"true"`)
+* `autoscroll`: whether to autoscroll to the output when prediction has finished (by default `"false"`)
+* `eager`: whether to load the Gradio app as soon as the page loads (by default `"false"`)
+
+Here's an example of how to use these attributes to create a Gradio app that does not lazy load and has an initial height of 0px. 
+
+```html
+&lt;gradio-app space="gradio/Echocardiogram-Segmentation" eager="true" initial_height="0px">&lt;/gradio-app>
```

Comment:
lost my mind trying to figure out why the page's width css was broken. turns out it was because we don't handle long lines in codeblocks correctly. Anyway, can you change this to:
```
&lt;gradio-app space="gradio/Echocardiogram-Segmentation" eager="true" 
initial_height="0px">&lt;/gradio-app>
```

Reply:
Weird, thanks for catching